doc/source/configuration/ci-cd.rst

```diff
@@ -181,3 +194,195 @@ Sometimes the kayobe docker image must be rebuilt the reasons for this include b
     * Update kolla-ansible
     * UID/GID collision when deploying workflows to a new environment
     * Prior to deployment of new a OpenStack release
+
+GitLab Pipelines
+================
+
+To enable CI/CD where GitLab Pipelines is used please follow the steps described below starting with the deployment of the runners.
+
+Runner Deployment
+-----------------
+
+1. Identify a suitable host for hosting the runners.
+    Ideally an infra-vm would be deployed to allow for easily compartmentalising the runners from the rest of the environment.
+    8 VCPUs and 16GB of RAM is recommended for the guest machine however this may need to be adjusted for larger deployments.
+    Whether the host is in an infra-vm or not it will need access to the :code:`admin_network` or :code:`provision_oc_network`, :code:`public_network` and the :code:`pulp registry` on the seed.
+    The steps will assume that an infra-vm will be used for the purpose of hosting the runners.
```

Comment:
I'd personally be in favor of adding a default config to our base environment which has some standard settings for cpus, memory, disk, runner name etc. Maybe we could even pre-configure the network interfaces to some extent

Reply:
Good idea could be done in follow PR.